packages/config/README.md

```diff
@@ -19,26 +19,27 @@ Please find a list of the default options below. They can be set in your project
+| `locations`     | `Array<string>`            | One or more [application shell](https://developers.google.com/web/fundamentals/architecture/app-shell) locations, e.g. `"/"` and/or `"/profile"`                                                                                                                                                                                                                                                                                                                |
+| `basePath`      | `String`                   | Root path of your application on the server. This is inserted as an HTTP path prefix between `host` and `locations`. For example, if you set e.g. `"basePath": "my-custom-basepath"` and `"location": ["/profile"]`, then your application would be available at `http://0.0.0.0:8080/my-custom-basepath/profile/`                                                                                                                                              |
+| `assetPath`     | `String`                   | HTTP root path of your asset files on the server. It's not dependent on `basePath`, you can define totally separate directories                                                                                                                                                                                                                                                                                                                                 |
+| `browsers`      | `String`                   | [Browserslist](https://www.npmjs.com/package/browserslist) query. Defaults to `"> 1%, last 2 versions, Firefox ESR"` (which is also the Browserlist default query). It is used for Babel and PostCSS                                                                                                                                                                                                                                                            |
+| `node`          | `String`                   | [Node target version](https://www.npmjs.com/package/babel-preset-env#targetsnode). Defaults to `current`. It is used to specify the Node version which Babel should compile for                                                                                                                                                                                                                                                                                 |
+| `envVars`       | `{ HOPS_MODE: 'dynamic' }` | Environment variables meant to be passed to [`webpack.EnvironmentPlugin`](https://webpack.js.org/plugins/environment-plugin/), i.e. made accessible to your app                                                                                                                                                                                                                                                                                                 |
```

Comment:
`{ HOPS_MODE: 'dynamic' }` is not a type, but the default value and therefore should be mentioned as such in the 'Description'.
For the `type` field you could do something like: `{ [key: String]: String }` or similar.

Reply:
Also: Isn't `NODE_ENV` also worth mentioning in the description as default/special value?